compiler/optimizations/inferConstRefs.cpp

```diff
@@ -649,6 +649,7 @@ static bool inferRefToConst(Symbol* sym) {
         info->fnUses = se->symbolSymExprsNext;
 
         CallExpr* call = toCallExpr(se->parentExpr);
+        if (call->isPrimitive()) continue;
```

Comment:
Unfortunately I think we might need to look a bit closer to see what the issue is. Above, in ``ConstInfo::ConstInfo``, we set ``fnUses`` based on the uses of a ``FnSymbol``, so it seems kind of odd that its parent would be a primitive here.

Is the type of primitive consistent? If so, I'd prefer this kind of skip to be keyed off of that.

Also, I think it would probably be most appropriate to set ``isRefToConst = false`` rather than ``continue``.